torch/custom_class_detail.h

```diff
@@ -148,6 +148,10 @@ TORCH_API at::ClassTypePtr getCustomClass(const std::string& name);
 // is a custom C++ class, otherwise return false.
 TORCH_API bool isCustomClass(const c10::IValue& v);
 
+// This API is for testing purposes ONLY. It should not be used in
+// any load-bearing code.
+TORCH_API std::vector<std::unique_ptr<jit::Function>>& customClassMethods();
```

Comment:
This should return the minimum necessary, which is just the schemas not the functions themselves. It should also have the language that it is for debug only in the function name so it will get caught in review `schemaOfCustomClassesForBCCheck()`